lib/go-tc/server_capability.go

```diff
@@ -0,0 +1,9 @@
+package tc
```

Comment:
woops, this file needs the apache header still

Reply:
Based on the file naming convention in this PR, should this file be named `server_server_capability.go`?